substrate/client/network/Cargo.toml

```diff
@@ -59,7 +59,7 @@ sp-blockchain = { workspace = true, default-features = true }
 sp-core = { workspace = true, default-features = true }
 sp-runtime = { workspace = true, default-features = true }
 wasm-timer = { workspace = true }
-litep2p = { workspace = true }
+litep2p = { workspace = true, features = ["websocket"] }
```

Comment:
nit: It doesn't look we require litep2p without `websocket` support in any `Cargo.toml` files. May be this should go to the workspace?